src/Sylius/Behat/Context/Api/Admin/ManagingPromotionCouponsContext.php

```diff
@@ -140,6 +204,15 @@ public function iAddIt(): void
         $this->client->create();
     }
 
+    /**
+     * @When I (try to) generate it
+     * @When I (try to) generate these coupons
+     */
+    public function iResetIt(): void
+    {
+        $this->client->executeCustomRequest($this->request);
+    }
```

Comment:
```suggestion
     * @When I (try to) generate it
     * @When I (try to) generate these coupons
     */
    public function iGenerateTheseCoupons(): void
    {
        $this->client->executeCustomRequest($this->request);
    }
```
Let's also change the only instance of `generate it` to `generate these coupons` in the `Generating new coupons with a large long code length value` scenario since it's the only usage of this wording